### PR TITLE
fix: contacts cannot be displayed in mobile

### PR DESCRIPTION
**Pull request by lelenei**
Set the height of the phone according to CSS media max-width

### CODE REVIEW

File: packages/maskbook/src/extension/options-page/DashboardRouters/Contacts.tsx [hunk 1]
```diff
@@ -151,32 +177,34 @@ export default function DashboardContactsRouter() {
...
+                                    if (isPagePending) return
+                                    if (data.visibleStopIndex === data.overscanStopIndex) nextPage()
+                                }}
+                                itemSize={64}
+                                itemCount={items.length}
+                                {...sizeProps}>
+                                {({ index, style }) =>
+                                    items[index] ? (
+                                        <ContactLine
+                                            style={style as any}
```
Comment (septs) [inline packages/maskbook/src/extension/options-page/DashboardRouters/Contacts.tsx:196]: ??????

File: packages/maskbook/src/extension/options-page/DashboardRouters/Contacts.tsx [hunk 2]
```diff
@@ -35,7 +35,33 @@ const useStyles = makeStyles((theme) =>
                 marginLeft: theme.spacing(-2),
                 marginRight: theme.spacing(-2),
             },
+            display: 'flex',
         },
+        fixed: {
+            height: '100%',
+            width: '100%',
+            ['@media (max-width: 320px)']: {
```
Comment (Jack-Works) [inline packages/maskbook/src/extension/options-page/DashboardRouters/Contacts.tsx:43]: hmmm, this section seems weird cc @lelenei can you explain?

Comment (lelenei) [reply packages/maskbook/src/extension/options-page/DashboardRouters/Contacts.tsx:43]: The height of the mobile phone screen cannot be the maximum, so the height of the mobile phone is set according to the width

Comment (lelenei) [reply packages/maskbook/src/extension/options-page/DashboardRouters/Contacts.tsx:43]: Height: '100%' should be deleted

Comment (jk234ert) [reply packages/maskbook/src/extension/options-page/DashboardRouters/Contacts.tsx:43]: On mobile (in fact, when the viewport's width <= 600), the height of the contacts list on the contact page, is computed as 0.

Comment (lelenei) [reply packages/maskbook/src/extension/options-page/DashboardRouters/Contacts.tsx:43]: I think this page is not adapted for mobile phones, so I just manually calculated the required height.Of course this is incomplete.

**Comment by Jack-Works** [comment]
let me create a new fix for this

**Pull request closed without merging**